easybuild/easyconfigs/i/icc/icc-2016.1.150-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,31 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
+name = 'icc'
+version = '2016.1.150'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp_update%(version_minor)s.tgz']
+
+checksums = ['4b93b0ff549e6bd8d1a8b9a441b235a8']
+
+gcc = 'GCC'
```

Comment:
as pointed out by @ocaisa in #2214, this should be `GCCcore`, and `binutils` should be included as well as a dep